README.md

```diff
@@ -71,3 +75,10 @@ To reopen jupyter notebooks when opening a new command prompt, navigate to your
 ### 4. Running tutorials as standalone scripts
 Tutorials are only available as Jupyter notebooks. If you want to run a notebook as a standalone Python script you can do that. Open the notebook in JupyterLab and export it as an [Executable Script](https://jupyterlab.readthedocs.io/en/stable/user/export.html).
                                                       
+## Data Blogs
+
+Data blogs are stored in the folder [Data_Blog](https://github.com/GeoNet/data-tutorials/tree/main/Data_Blog). Within this is a
```

Comment:
and this link can be changed from
` [Data_Blog](./Data_Blog)`


(the `./` indicate it is a folder in the current repo)

Reply:
But it will be a folder in the current repo. Maybe I don't understand???
![Screenshot from 2023-07-12 13-27-25](https://github.com/GeoNet/data-tutorials/assets/10933207/237c7373-c323-48e3-9ded-76df7b00ed9b)

Reply:
my very bad english, sorry!

when you add URL links to a markdown in git, if the file/folder you're pointing to is in the same repository (in this case the `data` one), you can just format the link without adding the full url, but just `./` (that is similar to your "home" directory on a terminal)

for example, if you want to link to the blog_01... you should add
`./Data_Blog/blog_01_val/` . you can see that this works if you click the links in the original `data` repo :)